ladybug/src/main/resources/springIbisDebuggerAdvice.xml

```diff
@@ -104,6 +104,22 @@
 					and
 					args(message, session)
 					"
+				method="debugSenderWithForwardInputOutputAbort"
+				arg-names="message, session"
+			/>
+			<aop:around
+				pointcut=
+					"
+					execution(
+						*
+						org.frankframework.core.ISender.sendMessageOrThrow(
```

Comment:
So, we've missed this method in the Ladybug debugger advice, for a long time?

Reply:
Yea, I think since we moved from/to OutputSteaming support.